mailbox/api/src/main/java/org/apache/james/mailbox/MailboxManager.java

```diff
@@ -74,6 +78,23 @@
 
 public interface MailboxManager extends RequestAware, RightManager, MailboxAnnotationManager, SessionProvider {
 
+    char SQL_WILDCARD_CHAR = '%';
+    EnumSet<MailboxManager.MessageCapabilities> DEFAULT_NO_MESSAGE_CAPABILITIES = EnumSet.noneOf(MailboxManager.MessageCapabilities.class);
+    int MAX_ATTEMPTS = 3;
+    Duration MIN_BACKOFF = Duration.ofMillis(100);
+    RetryBackoffSpec RETRY_BACKOFF_SPEC = Retry.backoff(MAX_ATTEMPTS, MIN_BACKOFF);
+
+    static MailboxQuery.UserBound toSingleUserQuery(MailboxQuery mailboxQuery, MailboxSession mailboxSession) {
```

Comment:
Maybe this can be moved in MailboxQuery ?

Reply:
why?

Reply:
mailboxQuery.toUserBound(session) sounds nice no?